examples/rag-wikipedia-ollama/main.go

```diff
@@ -100,7 +109,9 @@ func main() {
 	// category), but we don't do that in this example.
 	start := time.Now()
 	log.Println("Querying chromem-go...")
-	docRes, err := collection.Query(ctx, question, 2, nil, nil)
+	// "nomic-embed-text" specific prefix (not required with OpenAI's or other models)
+	query := "search_document: " + question
```

Comment:
```suggestion
	query := "search_query: " + question
```
this should fix
> 🚧 Looking at the similarities, it worked better without the prefixes 🤔

Reply:
Oh wow I overlooked that, thank you! :+1: 

I'm committing your suggestion and then will run the example again to check the similarity values

Reply:
So the similarity to the query is even lower, but at least the difference between the similarities of the first vs the second most similar document is much bigger again. Seems good ✅

- Original: `0.723627` for the first document, `0.550584` for the second (difference 0.17)
- Wrong prefix: `0.704056` for the first document, `0.620080` for the second (difference  ~0.08)
- With the fix: `0.655357` for the first document, `0.504042` for the second (difference ~0.15)